modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -127,6 +129,75 @@
  */
 public class CommerceOrderContentDisplayContext {
 
+	public CommerceOrderContentDisplayContext(
```

Comment:
add the configurationProvider to the other constructor method

Reply:
Done